src/include/mallocMC/creationPolicies/Scatter.hpp

```diff
@@ -151,12 +151,21 @@ namespace mallocMC
 #if _DEBUG || ANALYSEHEAP
         public:
 #endif
-            static constexpr uint32 minChunkSize1 = 0x10;
-            static constexpr uint32 HierarchyThreshold = (pagesize - 2 * sizeof(uint32)) / 33;
-            static constexpr uint32 minSegmentSize = 32 * minChunkSize1 + sizeof(uint32);
+            /* HierarchyThreshold defines the larges chunk size which can be stored in a segment with hierarchy.
+             * 32 chunks can be stored without a on page bitmask, therefore a hierarchy is only useful if we store at
+             * least 33 chunks. For 33 chunks we need two bitmasks, each 32bit.
+             */
+            static constexpr uint32 HierarchyThreshold = (pagesize - 2u * sizeof(uint32)) / 33u;
+            /* Calculate minimal chunk size which can fill a page, this avoids that small allocations
+             * fragment the heap and increases the possibility that a small allocation can reuse an
+             * existing chunk.
+             * Each page can have 32x32 chunks. To maintain 32 chunks we need 32 bitmask on the page (each 32bit)
+             */
+            static constexpr uint32 minChunkSize = (pagesize - 32u * sizeof(uint32)) / (32u * 32u);
+            static constexpr uint32 minSegmentSize = 32u * minChunkSize + sizeof(uint32);
             static constexpr uint32 tmp_maxOPM
```

Comment:
Unrelated to the change, but please add a comment to this variable. (and/or rename it as well)

Reply:
I will provide more docs in a separate PR.